tests/resources/test-types.json

```diff
@@ -1996,6 +2368,7 @@
             "forVehicleSize": null,
             "forVehicleConfiguration": null,
             "forVehicleAxles": [
+              1,
```

Comment:
Should have trailer entry in "forVehicleType"

Reply:
Done.